menu/champion_select/champion_select.py

```diff
@@ -0,0 +1,70 @@
+import pygame
+import sys
+from menu.button.Button import Button
+from menu.window.window import Window
+
+
+class ChampionSelect(Window):
+    def __init__(self, width, height):
+        super().__init__("Champion Select", width, height)
+        self.champion_index = 0
+        self.BUTTON_ARROW_RIGHT = None
+        self.BUTTON_ARROW_LEFT = None
+        self.width = width
+        self.height = height
+        self.BACK_BUTTON = None
+        self.NEXT_BUTTON = None
+        self.CHAMPION_SELECT_MOUSE_POS = None
+
+    def print_champion_select_menu(self):
+        scale = 10
+        self.screen.blit(self.BACKGROUND_IMAGE, (0, 0))
```

Comment:
What is (0, 0)?

Reply:
It has to be like that to cover whole window with image